RELEASE.md

```diff
@@ -0,0 +1,9 @@
+# Release instructions
+
+The following steps should be performed each and every time a new release is made.
+
+1. Switch to the `main` branch in your local installation
+2. Run `git pull` to get the latest code changes
+4. Edit [CHANGELOG.md](CHANGELOG.md) following the [Keep a Changelog](https://keepachangelog.com) convention
+5. Run `git commit -m "Updates CHANGELOG" && git push`
```

Comment:
Do we want to use Conventional Commits for this (to sync up with platform)?

e.g.
```suggestion
5. Run `git commit -m "docs: update CHANGELOG" && git push`
```